Source/WebKit/WebProcess/GPU/graphics/RemoteResourceCacheProxy.cpp

```diff
@@ -71,7 +71,7 @@ inline static RefPtr<ShareableBitmap> createShareableBitmapFromNativeImage(Nativ
 {
     auto imageSize = image.size();
 
-    auto bitmap = ShareableBitmap::create(image.size(), { image.colorSpace() });
```

Comment:
Won't this cause images using non-sRGB colorspaces (e.g. DisplayP3) to get clamped to sRGB?